### PR TITLE
cards

### PR DESCRIPTION
**Pull request by Mandhularajitha**

### CODE REVIEW

File: cards.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Arial;
+    box-sizing: border-box;
+}
+
```
Comment (znjs) [inline cards.css:7]: suggestion: better to have it in a `reset.css` file and import here

File: cards.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+.product_image {
+    width: 100%;
+
+}
+
+.product_title {
+    font-size: 20px;
+    margin-bottom: 5px;
```
Comment (znjs) [inline cards.css:36]: use `rem`s for responsiveness

File: cards.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+}
+
+.product_price {
+    margin-bottom: 5px;
+}
+
+.product_price>small {
+    font-size: 16px;
```
Comment (znjs) [inline cards.css:45]: use `rem`s for responsiveness

File: cards.css [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.product_price>span {
+    font-size: 18px;
+}
+
+.product_rating {
+    display: flex;
+    margin-bottom: 20px;
```
Comment (znjs) [inline cards.css:54]: use `rem`s for responsiveness

File: cards.css [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    margin-right: 4px;
+
+}
+
+.product_button {
+    width: 100%;
+    height: 40px;
+    font-size: 18px;
+    cursor: pointer;
+    margin-top: 10px;
```
Comment (znjs) [inline cards.css:67]: use `rem`s for responsiveness

**Review by znjs — CHANGES_REQUESTED** [review]

**Review by Sindhu983 — APPROVED** [review]

**Review by MonikaJhingan — APPROVED** [review]
The code looks good to be merged.

**Pull request merged**